### PR TITLE
adds a long distance messaging terminal

### PR DESCRIPTION
**Pull request by buffyuwu**
## About The Pull Request
craftable messaging terminals that allow players to send messages to each other. one is a computer terminal, and the other is a pigeon carrier (for legion immersion)

made in large part based on the requests console

## Pre-Merge Checklist

- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added messaging terminal
/:cl:

### CODE REVIEW

**Comment by AdamElTablawy** [comment]
i

i don't think pidgeons exist anymore

**Comment by buffyuwu** [comment]
> i
> 
> i don't think pidgeons exist anymore

image

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -464,7 +464,7 @@ GLOBAL_LIST_EMPTY(allConsoles)
 	emergency = null
 	update_icon()
 
-/obj/machinery/requests_console/proc/createmessage(source, title, message, priority)
+/obj/machinery/requests_console/createmessage(source, title, message, priority)
```
Comment (hecksadecimal) [inline code/game/machinery/requests_console.dm:467]: Why this change in requests_console.dm?

Comment (buffyuwu) [reply code/game/machinery/requests_console.dm:467]: linters demanded the parent object have the proc. we don't use request consoles, but this also doesn't hinder their functionality in any way :)

just moves the proc up one level because linters:tm:

**Review by hecksadecimal — APPROVED** [review]

**Comment by hecksadecimal** [comment]
I can't speak for the lore friendliness of pigeons, that's someone else's job. But code-wise this works fine.
